decidim-initiatives/app/views/decidim/initiatives/initiatives/_form.html.erb

```diff
@@ -26,9 +25,7 @@
                   } %>
 <% end %>
 
-<%= form.select :scope_id,
-                @form.available_scopes.map { |scope| [translated_attribute(scope.scope_name), scope&.scope&.id] },
-                { disabled: !@form.state_updatable? } %>
+<%= form.select :scope_id, @form.available_scopes.map { |scope| [translated_attribute(scope.scope_name), scope.scope&.id] }, {} %>
```

Comment:
I think we change the behavior here, and is not what we desire. The scope is not being disabled because is missing an extra hash parameter. 

```
select(attribute, choices, options = {}, html_options = {})
```

```suggestion
<%= form.select :scope_id, @form.available_scopes.map { |scope| [translated_attribute(scope.scope_name), scope.scope&.id] }, {}, { disabled: !@form.state_updatable? } %>
```